tests/load_main_menu_test.py

```
@@ -116,16 +108,27 @@ def on_console(msg) -> None:
+            f.write(shared_page.content())
 
         print(
             f"Failure logs: artifacts/test_load_main_menu_failure_console_logs_{timestamp}.txt. Error: {e}"
         )
         raise
     finally:
+        # 1. Unregister console listener from the shared module page
+        try:
+            shared_page.remove_listener("console", on_console)
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)